gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://apicast.io/poolicy-v1/schema#manifest#",
+  "name": "oauth2 token introspection policy",
+  "description": 
+  ["This policy executes OAuth 2.0 Token Introspection",
+   "(https://tools.ietf.org/html/rfc7662) for every API calls."],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "introspectionUrl": {
```

Comment:
This does not match the use in the policy. Here it is `introspectionUrl` but there it is `introspection_url`. 
We plan to have JSON schema validation during Policy initialization to verify those cases in CI, but right now it is just manual process, sorry.